### ISSUE TITLE
so native dynamic type representations? 

### ISSUE THREAD

**Issue by faceyspacey** [ericelliott/rfx#15]
i think it is the new hotness. 

could we generate JSDOC for existing documentation generators to make use of?

Whats its ETA?

**Comment by jadbox** [comment]
+1 for JSDOC support

**Comment by ericelliott** [comment]
These are planned, but they should be separate modules. Before we can do this, we need to parse rtype.

The spec has been a bit of a moving target, but I feel like it's becoming stable enough to start building tools on top of. If either of you want to take a crack at a parser, I'd welcome a PR for the [rtype repo](https://github.com/ericelliott/rtype).

If either of you want to take a parsed rtype and generate JSDoc, that would be a very welcome tool, but I think DefinitelyTyped TypeScript files & Markdown API documentation generators would probably accomplish the same goals, and be more broadly useful?

Just a guess. =)

**Comment by ericelliott** [comment]
JSDOC support is not planned. Similar functionality specifically for rfx + rtype is planned. You're welcome to take a crack at it and PR. If you do, please open a new issue.
